integration-tests/src/test/java/tachyon/client/TachyonFileSystemIntegrationTest.java

```diff
@@ -153,8 +153,8 @@ public void getFileStatusTest() throws IOException, TException {
   public void mkdirTest() throws IOException, TException {
     String uniqPath = PathUtils.uniqPath();
     for (int k = 0; k < 10; k ++) {
-      Assert.assertTrue(sTfs.mkdirs(new TachyonURI(uniqPath + k)));
-      Assert.assertTrue(sTfs.mkdirs(new TachyonURI(uniqPath + k)));
+      Assert.assertTrue(sTfs.mkdirs(new TachyonURI(uniqPath + k), true));
```

Comment:
Define the following constant in TachyonFileSystem:

`` public static final RECURSIVE = true;``

and use it everywhere to improve the code readability:

`` Assert.assertTrue(sTfs.mkdirs(new TachyonURI(uniqPath + k), TachyonFileSystem.RECURSIVE));``